lib/money/money.rb

```diff
@@ -190,10 +191,10 @@ def to_s(style = nil)
     when :legacy_dollars, nil
       sprintf("%.2f", value)
     when :legacy_cents
-      (value * 100).round.to_i.to_s
-    when :minor_units
-      (value * currency.subunit_to_unit).round.to_i.to_s
-    when :major_units
+      cents.to_s
+    when :subunits
```

Comment:
> it's really more like total_in_subunits or total_in_lowest_unit
Not sure what you mean here. subunits are just like cents for 2 digit currencies
```
Money.new(1, 'USD').cents == 100
Money.new(1, 'USD').subunits == 100
#But
Money.new(1, 'JPY').cents == 100
Money.new(1, 'JPY').subunits == 1
```